ops/scripts/todo-checker.sh

```diff
@@ -109,7 +109,6 @@ for todo in $todos; do
 
     # Check issue state
     STATE=$(echo "$RESPONSE" | jq -r .state)
-
```

Comment:
Is there a ticket to fix this script?

Reply:
I am looking at it but good call that we should track that with an issue.  Created https://github.com/ethereum-optimism/optimism/issues/14635